src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let count = 0;
+
+  if (days < 3) {
+    count = days * 40;
```

Comment:
What are 40 and 3? Let's create variables for all the main values, so your code will be more clear and understandable

Reply:
For 40 i've made a const pricePerDay = 40; but the 3 number is the parameter of the function with a if function so i cannot make a variable for the parameter. Make sense? If not i have to write a function if days < 3 equal a variable daysLessThanThree? 

I try to commit the new resolution in the main time. Thanks for your time